csharp/NegativePatterns.cs

```diff
@@ -4,40 +4,29 @@ namespace FeatureFlags
     public class NegativePatterns 
     {
         private readonly IFeatureService _featureService;
-        private readonly bool _isOn;
 
         public NegativePatterns(IFeatureService featureService)
         {
             _featureService = featureService;
 
             // Do not store evaluated feature flags. 
             // You have no way of knowing the lifecycle management of this class
-            _isOn = _featureService.Evaluate("flag-name", user);
         }
 
         // This is not the worst, but it makes life a little more complicated when it comes to feature flag removal.
         public void PerformCode(string user)
         {
-            if (_featureService.Evaluate("flag-name", user)){
-                // New code
-            } else {
-                // Old code
-            }
+            // New code
```

Comment:
Depending on the diffing capabilities, it might get a bit nasty because of the change here.